esmvalcore/_config.py

```diff
@@ -38,13 +38,12 @@ def read_config_user_file(config_file, recipe_name):
         'write_netcdf': True,
         'compress_netcdf': False,
         'exit_on_warning': False,
-        'max_data_filesize': 100,
-        'output_file_type': 'ps',
-        'output_dir': './output_dir',
-        'auxiliary_data_dir': './auxiliary_data',
+        'output_file_type': 'png',
+        'output_dir': 'esmvaltool_output',
+        'auxiliary_data_dir': 'auxiliary_data',
         'save_intermediary_cubes': False,
-        'remove_preproc_dir': False,
-        'max_parallel_tasks': 1,
+        'remove_preproc_dir': True,
```

Comment:
I would keep `False` as a default.
For a first-time user it may be more convenient to get the full output.

Reply:
I think that's only convenient if you're going to develop a diagnostic, not if you're a first time user.

Reply:
Anyway, the default has been true in all the example configuration files we provide for some time now, so this pull request just aligns the defaults in the code with those in the example files.

Reply:
Yes, but the [documentation](https://esmvaltool.readthedocs.io/en/latest/getting_started/outputdata.html) says that the output directory will contain 4 subdirs, including `preproc/`. It could be confusing if `preproc/` is deleted by default.

Reply:
If you scroll down a bit in that documentation, it says that the directory is deleted by default. I've updated the section to make it more clear.